NameGame/NameGame/UI/Game/GameViewModel.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GameViewModel.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/27/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct GameViewModel {
+    
+    let choices: [UserRecord]?
+    let correctChoice: UserRecord?
```

Comment:
Add the images to this model. Also, we should make these properties non-optional.